src/Forge.php

```diff
@@ -135,16 +135,19 @@ public function create()
     /**
      * Returns single server.
      *
-     * @param int $serverId
+     * @param int     $serverId
+     * @param boolean $reload   (optional) indicates whether the server should be reloaded
      *
      * @return \Laravel\Forge\Server
      */
-    public function get(int $serverId)
+    public function get(int $serverId, bool $reload = null)
     {
-        if ($this->lazyLoadInitiated() && isset($this->serversMap[$serverId])) {
-            return $this->items[$this->serversMap[$serverId]];
-        } elseif (isset($this->serversCache[$serverId])) {
-            return $this->serversCache[$serverId];
+        if (!$reload) {
```

Comment:
I have corrected this, but presumably I need to leave the default 
```
return $this->loadSingleServer($serverId);
```
at the end of the function too?